src/home-page/home-page.component.html

```diff
@@ -12,14 +12,15 @@ <h2>City</h2>
     <p class="col">Temp</p>
     <p class="col">Humidity</p>
   </div>
-  <div *ngFor="let city of cities" class="cityList row">
-    <a [routerLink]="['/home', city.name]">
-      <p class="col">{{city.name}}</p>
-      <p class="col-4">{{city.weather[0].description}} <img
-        src="http://openweathermap.org/img/wn/{{city.weather[0].icon}}.png"/></p>
-      <p class="col">{{city.main.temp}}</p>
-      <p class="col">{{city.main.humidity}}%</p>
-    </a>
+  <div *ngIf="isLoaded">
```

Comment:
even better if you implement some kind of spinner to display when isLoaded is false 👍